components/epaxos/src/instance/instance.rs

```diff
@@ -0,0 +1,32 @@
+use super::super::command;
+use super::super::replica;
+
+#[cfg(test)]
+#[path = "./tests/instance_tests.rs"]
+mod tests;
+
+/// protocol buffer serialized
+pub type InstanceNum = i64;
+pub struct InstanceID {
+    pub replica_id: replica::ReplicaID,
+    pub instance_num: InstanceNum,
```

Comment:
```suggestion
    pub num: InstanceNum,
```
已经在一个InstanceID结构体了, 感觉不需要再携带instance信息了

Reply:
也行，就叫num吧